### ISSUE TITLE
firstqr log missing in latest version

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2435]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.30.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.30.0...4.30.1) (2022-01-25)

- Release 4.30.1 [`1ed3194`](https://github.com/open-wa/wa-automate-nodejs/commit/1ed319462447d5e79abf6a14e03c472166fd3cca)
- 🐛 fix missing firstqr log #2435 [`c890d6d`](https://github.com/open-wa/wa-automate-nodejs/commit/c890d6d9b96d3c291448643d02544da16ed53c85)
- updated types-only package [`91c6c33`](https://github.com/open-wa/wa-automate-nodejs/commit/91c6c33cf924ec5d2eb841d1b39899bb34238c86)
